client/my-sites/domains/components/domain-warnings/pending-gapps-tos-notice.jsx

```diff
@@ -61,50 +65,63 @@ const PendingGappsTosNotice = React.createClass( {
 		}
 
 		return 'info';
-	},
+	}
 
 	getExclamation( severity ) {
+		const { translate } = this.props;
 		const translationOptions = {
 			context: 'Beginning of Gapps pending account notice',
 			comment: 'Used as an exclamation in Gapps\' pending account notice'
 		};
 
 		switch ( severity ) {
 			case 'warning':
-				return this.props.translate( 'Attention!', translationOptions );
+				return translate( 'Attention!', translationOptions );
 
 			case 'error':
-				return this.props.translate( 'Urgent!', translationOptions );
+				return translate( 'Urgent!', translationOptions );
 
 			default:
-				return this.props.translate( 'You\'re almost there!', translationOptions );
+				return translate( 'You\'re almost there!', translationOptions );
 		}
-	},
-
-	generateLogInClickHandler( { domainName, user, severity, isMultipleDomains } ) {
-		return () => {
-			this.recordEvent( 'pendingAccountLogInClick', {
-				domainName,
-				isMultipleDomains,
-				severity,
-				user,
-				section: this.props.section,
-				siteSlug: this.props.siteSlug
-			} );
-		};
-	},
+	}
 
-	generateFixClickHandler() {
-		return () => {
-			this.recordEvent( 'fixPendingEmailSiteNoticeClick', this.props.siteSlug );
-		};
-	},
+	recordLogInClick = ( domainName, user, isMultipleDomains ) => {
+		this.props.pendingAccountLogInClick( {
+			domainName,
+			isMultipleDomains,
+			user,
+			severity: this.getNoticeSeverity(),
+			section: this.props.section,
+			siteSlug: this.props.siteSlug
+		} );
+	};
+
+	generateLogInClickHandlerOneDomain = () => {
```

Comment:
This method's name does not reflect what it does anymore - it does not generate a click handler, it _is_ a click handler :)